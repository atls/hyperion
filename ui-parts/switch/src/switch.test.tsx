import React      from 'react'
import { render } from '@testing-library/react'

describe('Switchh component', () => {
  describe('snapshots', () => {
    it('should match latest render snapshot', () => {
      const { asFragment } = render(<div />)

      expect(asFragment()).toMatchSnapshot()
    })
  })
})
