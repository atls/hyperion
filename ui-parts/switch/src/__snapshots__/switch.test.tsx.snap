// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switchh component snapshots should match latest render snapshot 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
